src/main/scala/org/scalacheck/ops/package.scala

```diff
@@ -1,10 +1,11 @@
 package org.scalacheck
 
+import org.scalacheck.ops.time.JavaTimeImplicits
 import org.scalacheck.ops.time.joda.JodaTimeImplicits
 
 /**
  * @note [[JodaTimeImplicits]] are included as well, since this library is pretty much the de-facto
  *       date / time library for Java / Scala. This may change in future versions if this is
  *       somehow no longer the case.
  */
-package object ops extends ScalaCheckImplicits with JodaTimeImplicits
+package object ops extends ScalaCheckImplicits with JodaTimeImplicits with JavaTimeImplicits
```

Comment:
So you're forcing the caller to have both java and joda? It'd be better to have two separate artifacts, one with joda, one with java